docusaurus.config.js

```diff
@@ -287,34 +287,5 @@ module.exports = {
         },
       },
     ],
-    [
```

Comment:
What was this?

Reply:
Yeah it was pulling in some READMEs from the chain repo and updating some links, but the final document looks exactly the same with and without it, so I just removed it.

Reply:
I should ask what was it intending to do

Reply:
Then my previous comment still answers your question, I think 🤔